english/coffee/nude-run/warm-water/warm-water.md

```diff
@@ -6,3 +6,12 @@ at last speaks again.
 
 "You know," says the badger, "the more I think about it, the more certain
 I become that we are all made of bees."
+
+He says this with the absurd certainty of Yoda, or the Little Prince. Part of
+you wants to scream (there is no such thing as speaking badgers, and I am
+certainly not made of bees). But there is also a small part of you that feels
+the pull of the badgers soft voice, his calm but engaging eyes.
+
+Do you:
+* [rebel against the absurdity of it all?] (rebel/rebel.md)
+* [let your mind surrender] (surrender/surrender.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.